apps/webapp/web/static/js/index.js

```diff
@@ -34,17 +34,24 @@ function requireAuth(nextState, replace) {
   }
 }
 
+class Application extends React.Component{
+  render(){
+    const env = Auth.getEnvironment();
+    return (<Router history={browserHistory}
+      render={applyRouterMiddleware(useRelay)}
+      environment={env}>
```

Comment:
I think `environment={Auth.getEnvironment()}` would be better ... I don't know why suddenly I completely missed where `env` came from.